osrf_pycommon/cli_utils/verb_pattern.py

```diff
@@ -17,7 +17,7 @@
 import sys
 import inspect
 
-import pkg_resources
+import importlib_metadata
```

Comment:
This syntax targets the backport package exclusively, and needs to be updated to handle the Python 3.8+ built-in module.

```suggestion
try:
    import importlib.metadata as importlib_metadata
except ModuleNotFoundError:
    import importlib_metadata
```

Reply:
Good call.  Fixed in 9e2ffde.